tools/midi2piano/midi2piano/obj/x86/Debug/midi2piano.csproj.CoreCompileInputs.cache

```diff
@@ -0,0 +1 @@
+d9f161e9d0130fbebe72002a19544a2a8fae236d
```

Comment:
Is this an intended change?

Reply:
this file should be in .gitignore lol

Reply:
they are not covered by my gitignore :|
```
#ignore misc BYOND files
Thumbs.db
*.log
*.int
*.rsc
*.dmb
*.lk
*.backup
*.dmm.before
data/
cfg/
tmp/
tools/Runtime Condenser/Input.txt
tools/Runtime Condenser/Output.txt
tools/MapDaemon/logs/*.txt
.vscode/*
```

Reply:
I meant they should be but weren't so I added it in a different pr